__tests__/cache/SimpleCache.spec.js

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { createStore } from 'redux';
+import SimpleCache from '../../src/cache/SimpleCache';
+import * as actions from '../../src/cache/actions';
+
+const fakeError = new Error('Fake Error');
+const fakeData = {
+  status: 200,
+  ok: true,
+  body: {
+    fakeData: true,
+  },
+};
+
+const createScenario = (dispatch, actionKeys, hash) => {
+  const actionOrder = actionKeys.map((key) => actions[key]);
+  actionOrder.forEach((actionCreator) => {
+    dispatch(
+      actionCreator({
+        hash,
+        error: fakeError,
+        value: fakeData,
+      })
+    );
+  });
+};
+
+describe('SimpleCache', () => {
+  let store;
+  const cache = SimpleCache();
+  beforeEach(() => {
+    store = createStore(cache.reducer, cache.reducer(undefined, { type: '' }));
+  });
+  it('should reflect load state', () => {
+    const { dispatch, getState } = store;
+    createScenario(dispatch, ['loadingAction'], 'abc1234');
+    expect(getState()).toMatchInlineSnapshot(`
+      Object {
+        "data": Object {},
+        "errors": Object {},
+        "loading": Object {
+          "abc1234": "abc1234",
```

Comment:
will the value be set to a boolean or the same key value?

Reply:
`getCacheByKey` would have that answer. I think I missed a `!!` for https://github.com/americanexpress/fetchye/pull/8/files#diff-f37b4cd9154fbb3966600040adb4870eR97

Reply:
Fixed in https://github.com/americanexpress/fetchye/pull/8/commits/b7a24b8c48335a0bec2bc15e611d2af45dd4defa